client/App.jsx

```diff
@@ -12,7 +13,37 @@ class App extends React.Component {
     this.state = {
       totalScore: null,
       totalReviews: 0,
-      sortedValue: 'highestRating'
+
+      sortedValue: 'highestRating',
+      reviewsArray: [],
+      showReviews: 3
+    };
+  }
+
+  componentDidMount() {
+    $.ajax({
+      url: '/api',
+      method: 'GET',
+      success: (res) => {
+        this.setState({
+          totalScore: this.state.totalScore,
+          totalReviews: this.state.totalReviews,
+          sortedValue: this.state.sortedValue,
+          reviewsArray: res,
+          showReviews: this.state.showReviews
+        });
+        console.log(this.state.reviewsArray);
+      }
+    })
+  }
+
+  loadMore() {
+    this.setState({
+      totalScore: this.state.totalScore,
+      totalReviews: this.state.totalReviews,
+      sortedValue: this.state.sortedValue,
+      reviewsArray: this.state.reviewsArray,
+      showReviews: this.state.showReviews += 3
```

Comment:
It seems like the only state you're changing is showReviews. Is there a reason you're setting all of these other states to equal themselves? You can target a single state with setState and the other values should remain the same. Definitely let me know if I'm missing something here, though.